### ISSUE TITLE
2:4 sparse retraining lowers the accuracy of the network

### ISSUE THREAD

**Issue by nralka2007** [NVIDIA/apex#1550]
I pruned a UNet with 2:4 sparsity and retrained using ASP library. the accuracy seems to drop after retraining. The parameters used for retraining are the same as the ones used to train the network from scratch. Any guidelines to get back the accuracy of the network after pruning?

**Comment by rg321** [comment]
try to finetune from dense pre-trained checkpoint. in my observation, its generally hard to regain original accuracy fully if training from scratch
